lib/src/controls_column.dart

```diff
@@ -60,6 +61,8 @@ class ControlsColumn extends StatelessWidget {
               onPressed: isNavigatingActive
                   ? null
                   : () => onControlModeChanged(FeedbackMode.navigate),
+              disabledTextColor:
+                  FeedbackTheme.of(context).activeFeedbackModeColor,
```

Comment:
@ueman Huh the previous code worked in the example app. Oh well, either way it seems better to put this in `FeedbackTheme`. updated!